api/leaderboard.go

```diff
@@ -514,18 +506,12 @@ func (app *App) GetAroundScore(ctx context.Context, req *api.GetAroundScoreReque
 	if tenantID != nil {
 		result, err := app.Enricher.Enrich(ctx, tenantID[0], req.LeaderboardId, members)
```

Comment:
Instead of having line 514, what about this?

```suggestion
		members, err := app.Enricher.Enrich(ctx, tenantID[0], req.LeaderboardId, members)
```